### DISCUSSION TITLE
DocBin Attributes

### DISCUSSION THREAD

**Discussion by JoeIsaBro** [explosion/spaCy#12668] [Help: Other Questions] [answered]
I'm trying to load 700k documents into a docbin to save to disk and load back at a later stage. After I load back the document, I need to extract all useful entities to add them to a knowledge graph. I've pasted my code below, I can't seem to be able to extract anything using the loadedDocBin.__getattribute__ or doc.ents. 

Can someone explain to me what I'm doing wrong? I think I'm missing a fundamental point here, just can't figure out what...

```
FRdoc_bin = DocBin ()
doc = frNLP(text)
FRdoc_bin.add(doc)
FRdoc_bin.to_disk(CreatedModelPath+r'\FRdocBin'+'.nlp')
```

```
import spacy
from spacy.tokens import DocBin
from spacy.vocab import Vocab

nlp = spacy.load('fr_dep_news_trf')
DocBinPath = r'C:\[Redacted]\FRdocBin.nlp'

loadedDocBin = DocBin(Vocab()).from_disk(DocBinPath)  

DocList=list(loadedDocBin.get_docs(nlp.vocab))
for doc in DocList
    People = list(set([ent.text for ent in doc.ents if ent.label_=='PERSON']))    #doc.ents outputs empty
```

Let me know if anything doesn't make sense or if I can provide anything else :)

**Comment by adrianeboyd** [comment, marked as answer]
If you're using `fr_dep_news_trf` as `frNLP` in the first step, then there are no entities annotated in your documents because this pipeline doesn't include NER?

**Reply by JoeIsaBro** [reply]
Thank you... I know I was doing SOMETHING dumb. I had to change the model to a less accurate one but it works :)
